pilates/utils/geog.py

```diff
@@ -187,25 +213,9 @@ def get_zone_from_points(df, zones_gdf, zone_id_col, local_crs):
 
     # Spatial join
     intx = gpd.sjoin(
-        gdf.reset_index(), zones_gdf.reset_index(),
+        gdf, zones_gdf.reset_index(),
         how='left', op='intersects')
+    
+    
 
-    # Drop duplicates and keep the one with the smallest H3 area
-    intx['intx_area'] = intx['geometry'].area
-    intx = intx.sort_values('intx_area')
-    intx.drop_duplicates(subset=[gdf.index.name], keep='first', inplace=True)
-    intx.set_index(gdf.index.name, inplace=True)
-    gdf[zone_id_col] = intx[zone_id_col].reindex(gdf.index)
```

Comment:
I dropped this functionality because in H3 it was possible to have overlapping zones (e.g. different HEX resolutions). While I don't expect for the zone_gdf to have overlapping geometries, I did added the assertion `len(intx) == len(gdf)` which verifies that one point is not assigned to 2+ zones.

Reply:
Do you need to push this commit? I'm not seeing it in the PR

Reply:
just push it. Check change line 219